src/wp-admin/js/inline-edit-post.js

```diff
@@ -100,27 +148,36 @@ inlineEditPost = {
 			}
 		});
 
+		// If no boxes where checked, just hide the quick/bulk edit rows.
 		if ( c ) {
 			return this.revert();
 		}
 
+		// Add onclick events to the delete-icons for the post titles.
```

Comment:
maybe you should try the bulk editing and see how this looks. That makes everything a lot more easy to understand. If you see the image, is it still unclear?

![pages_ _local_wordpress_dev_ _wordpress](https://cloud.githubusercontent.com/assets/7293908/17973735/dc454292-6ae3-11e6-89e9-177ff01b656b.jpg)